CHANGES.md

```diff
@@ -11,6 +11,9 @@ Bugfix 🐛:
  - Space Invite by link not always displayed for public space (#3345)
  - Wrong copy in share space bottom sheet (#3346)
  - Fix a problem with database migration on nightly builds (#3335)
+ - Spaces | Personal spaces add DM - Web Parity (#3271)
+ - Spaces | Improve 'Leave Space' UX/UI (#3359)
+ - Don't create private spaces with encryption enabled (#3363)
```

Comment:
It should be better to include one changelog new line along with each dedicated commit.
It will help when annotating this file to see the related change.
Any way this will be probably not possible anymore if we start using towncrier...

Reply:
thx, noted